TicketMate.Persistence/DataRequestObjects/ProjectRequests/UpdateProjectByGuid.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TicketMate.Persistence.DataRequestObjects.ProjectRequests
+{
+    public class UpdateProjectByGuid : IDataExecute
+    {
+        public Guid Guid { get; set; }
+
+        public string? Name { get; set; }
+
+        public bool? IsActive { get; set; }
+
+        public UpdateProjectByGuid(Guid guid, string? name, bool? isActive) 
+        {
+            Guid = guid;
+
+            if (name == "") name = null;

```

Comment:
can we set name to null if string.IsNullOrWhiteSpace(name) ?

Reply:
For sure! I'd love to hear more on this today at standup.